tests/tenant-fetcher-job/tests/handler_test.go

```diff
@@ -75,11 +75,11 @@ const (
 		"guid": "%s",
 		"displayName": "%s",
 		"subdomain": "%s",
-		"parentGuid": "%s",
```

Comment:
Leave it as well with a different value from the globalAccountGUID field to assert that the feature actually works.